[2024-12-16T16:19:02.041+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-16T16:19:02.049+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-16T16:19:02.056+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-16T16:19:02.056+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-16T16:19:02.063+0000] {taskinstance.py:2889} INFO - Executing <Task(TriggerDagRunOperator): trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2> on 2024-12-15 00:00:00+00:00
[2024-12-16T16:19:02.067+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=279) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-16T16:19:02.067+0000] {standard_task_runner.py:72} INFO - Started process 282 to run task
[2024-12-16T16:19:02.067+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'AF_TEST1', 'trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2', 'scheduled__2024-12-15T00:00:00+00:00', '--job-id', '1930', '--raw', '--subdir', 'DAGS_FOLDER/script/AF_TEST1.py', '--cfg-path', '/tmp/tmpy9wvxum8']
[2024-12-16T16:19:02.068+0000] {standard_task_runner.py:105} INFO - Job 1930: Subtask trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2
[2024-12-16T16:19:02.089+0000] {task_command.py:467} INFO - Running <TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-15T00:00:00+00:00 [running]> on host 764f0023e79c
[2024-12-16T16:19:02.129+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test-framework' AIRFLOW_CTX_DAG_ID='AF_TEST1' AIRFLOW_CTX_TASK_ID='trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T00:00:00+00:00'
[2024-12-16T16:19:02.129+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-16T16:19:04.410+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:175 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.410+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.412+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1151 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.412+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.412+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1271 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.413+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1406 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.413+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.413+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1495 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.414+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.414+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1750 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.414+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.415+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2110 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.415+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.416+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2468 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.418+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:3987 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.418+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.418+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4127 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.419+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4186 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.419+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4225 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.419+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.420+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5209 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.421+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5265 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.421+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5302 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.421+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5358 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6437 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6538 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.424+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.424+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7023 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7267 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7333 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7378 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.426+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7440 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.427+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.427+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.427+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.428+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8368 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.429+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9475 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.429+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9531 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.431+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.431+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.431+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10625 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.431+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10680 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.432+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10923 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.432+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10979 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.433+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11016 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.433+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11072 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.433+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11231 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.434+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.434+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11420 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.434+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11477 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.435+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12300 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.436+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12383 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.436+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.437+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12524 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.437+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.438+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.438+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13139 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.438+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13200 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.439+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13886 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.439+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13941 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13977 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14026 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14318 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14388 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14436 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14492 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.442+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14744 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.442+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14810 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.443+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.443+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15482 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.443+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15527 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.444+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15578 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.444+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15897 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.445+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15953 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16712 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16780 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.447+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.447+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16988 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.448+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.448+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.448+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17228 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.449+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17284 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.452+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20234 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.453+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20289 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.453+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20862 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.454+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20921 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.454+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20960 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.454+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.456+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.456+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21377 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.457+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21418 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.457+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21478 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.457+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.458+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21668 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.459+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22155 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.459+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22218 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22559 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22626 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22671 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.461+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.461+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23061 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.462+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23128 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.462+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23173 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.462+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23224 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.463+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23511 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.463+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23567 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.464+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23968 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.464+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:24018 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.466+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25343 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.466+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.466+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25434 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.467+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.468+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26723 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.468+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.469+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27121 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.469+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27177 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.470+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27461 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.470+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27517 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.471+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28053 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.471+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28109 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28536 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28592 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.473+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.473+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28982 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.476+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30727 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.476+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30783 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.477+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31296 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.477+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31344 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.478+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.478+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31673 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.479+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32120 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.479+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32176 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.479+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32339 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32395 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32432 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.482+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33502 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.482+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33558 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.482+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33869 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.483+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33928 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.483+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33967 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.483+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34021 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.484+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34739 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.485+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34806 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.485+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34852 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.485+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34908 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36915 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36971 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.489+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37417 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.489+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37473 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.490+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37845 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.490+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37901 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.491+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38208 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.491+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38264 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.491+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38301 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.491+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38351 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.492+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38690 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.492+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.493+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39141 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.493+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39192 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.494+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39419 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.494+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.495+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.495+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39738 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.496+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.496+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40084 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.497+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41138 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.498+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.498+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41359 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.498+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.499+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.499+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41507 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.499+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41875 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.500+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41947 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.504+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.504+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.504+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42534 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.504+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.505+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42627 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.505+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42692 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.506+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43020 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.506+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43079 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.506+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43118 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.507+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43168 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.507+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43514 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.508+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43573 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.508+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.508+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43666 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.509+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43989 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.509+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44045 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.510+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44386 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.510+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44442 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.514+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.518+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45058 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.523+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45444 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.525+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.526+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46042 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.527+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46098 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.531+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46702 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.531+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.533+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47000 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.534+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47050 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.538+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47648 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.539+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47704 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.541+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48113 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.544+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48190 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.545+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48244 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.547+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48321 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.551+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.553+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49555 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.556+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.557+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50002 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.558+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50052 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.560+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50107 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.561+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50379 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.562+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50449 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.562+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.562+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50554 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.563+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.563+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.563+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.564+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50770 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.564+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51046 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.565+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51116 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.565+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51166 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.566+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51226 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.566+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.567+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51540 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.567+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51787 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.567+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51851 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.568+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51895 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.568+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51958 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.569+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52212 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.569+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.569+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52463 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.570+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.571+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53040 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.571+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53111 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.571+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53334 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.572+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.573+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53961 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.573+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54059 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.573+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.574+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.575+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55022 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.575+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55078 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.576+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55310 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.576+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55365 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.577+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.577+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.578+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56508 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.578+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56564 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.579+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.579+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57147 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.580+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57187 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.580+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57237 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.581+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57560 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.581+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57620 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.581+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.581+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.583+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59106 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.584+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59162 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.584+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.584+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59396 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.585+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.585+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59535 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.585+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59594 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.586+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59677 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.586+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59733 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.586+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59818 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.587+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60639 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.588+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.588+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60778 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.589+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60863 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.589+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61167 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.590+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61250 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.590+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61306 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.590+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61387 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.591+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61696 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.591+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61752 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.591+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61954 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.592+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61992 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.593+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.593+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62715 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.595+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63882 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.595+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63946 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.596+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64582 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.596+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.597+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64940 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.597+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.598+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65117 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.598+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65199 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.598+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65385 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.599+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.599+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65874 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.600+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65950 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.600+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.600+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.601+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66686 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.601+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.602+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66918 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.602+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66964 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.603+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67735 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.604+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67791 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.604+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68043 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.604+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68097 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.605+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.605+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-16T16:19:04.902+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-16T16:19:04.903+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=AF_TEST1, task_id=trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2, run_id=scheduled__2024-12-15T00:00:00+00:00, execution_date=20241215T000000, start_date=20241216T161902, end_date=20241216T161904
[2024-12-16T16:19:04.923+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-16T16:19:04.933+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-16T16:19:04.933+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
